lib/cli/test/unit/Cardano/CLISpec.hs

```diff
@@ -461,9 +469,26 @@ spec = do
             , expectedStdout = "Prompt: ******\ESC[1D \ESC[1D\ESC[1D \ESC[1D**\n"
             , expectedResult = "pata14" :: Text
             }
+    describe "initTracer" $ do
+        it "no settings" $ do
+            defaultConfig <- getCG <$> defaultConfigStdout
+            let fp = "../../specifications/logging/empty.yaml"
```

Comment:
**[change required]**

Use [`(</>)`](https://hackage.haskell.org/package/filepath-1.4.2.1/docs/System-FilePath-Posix.html#v:-60--47--62-) from `System.FilePath` instead of hard-coded separators. 

**Why**

Hard-coded path don't play well with cross-compiled code :s, so, always rely on platform-agnostic combinator like `(</>)` to combine paths.